doc_test.go

```diff
@@ -71,7 +71,7 @@ func ExampleEmailService_CreateAccount() {
 
 	// NOTE: All parameters except for EmailAccount and Password are optional and can be omitted.
 	createaccount, _ := client.Emails.CreateAccount(context.Background(), glesys.CreateAccountParams{
-		EmailAccount:       "new_user@example.com",
+		EmailAccount:       "new_user@your-domain.com",
```

Comment:
I rather have `example.com` than `your-domain.com`, not the most important thing in the world though.

Reply:
That makes sense, I will update it. `example.com` is already used in `ExampleServerService_Edit` so this change will make the domain used within the test consistent.